ietf/draft-ietf-calext-jscalendar-icalendar-09.txt

```diff
@@ -0,0 +1,6272 @@
+         This MUST be ICalProperty.
+
+      name: String (mandatory)  This is the name of the iCalendar
+         property in lowercase.
+
+      valueType: String (optional)  This is the name of the iCalendar
+         property value type in lowercase.
+
+      parameters: String[String] (optional)  This contains parameters of
+         the iCalendar property.  The value MUST comply with iCalendar
+         parameters formatted in jCal as defined in Section 3.5 of
+         [RFC7265].
+
+5.1.4.  locations (Participant)
+
+   Name:
+      locations
+
+
+
+Stepanek                   Expires 16 May 2025                [Page 101]
+
+Internet-Draft                 JSCalendar                  November 2024
+
+
+   Context:
+      Participant
+
+   Type:
+      Id[Location] (optional)
+
+   Description:
+      This represents locations associated with the participant.  In
+      contrast to the locationId property, this allows to associate
+      multiple locations with the Participant object, and to express
+      that a location just is relevant for this participant.
+
+5.1.5.  scheduleAgent (Event, Task)
+
+   Name:
+      scheduleAgent
+
+   Context:
+      Event, Task
+
+   Type:
+      String (optional, default "server")
+
+   Description:
+      This property represents the SCHEDULE-AGENT parameter [RFC6638]
+      (Section 7.1) specified on an ORGANIZER property.  In context of
+      scheduling as outlined in [RFC6638], a scheduling client can set
+      this property to indicate who is responsible for sending
+      scheduling messages to the organizer in the replyTo property.  The
+      same values as for the scheduleAgent property of the Participant
+      object [RFC8984] (Section 4.4.6) are allowed, the "none" value
+      indicates that no scheduling message is to be sent to the
+      organizer.  The property MUST NOT be preserved in the JSCalendar
+      object on the server or appear in a scheduling message.  If this
+      property is set, then the replyTo property MUST be set.
+
+5.1.6.  scheduleForceSend (Event, Task)
+
+   Name:
+      scheduleForceSend
+
+   Context:
+      Event, Task
+
+   Type:
+      Boolean (optional, default "false")
+
+   Description:
+
+
+
+Stepanek                   Expires 16 May 2025                [Page 102]
+
+Internet-Draft                 JSCalendar                  November 2024
+
+
+      This property represents the SCHEDULE-FORCE-SEND parameter
+      [RFC6638] (Section 7.2) specified on an ORGANIZER property.  In
+      context of scheduling as outlined in [RFC6638], a scheduling
+      client can set this property to true to request that the server
+      send a scheduling message to the organizer defined in the replyTo
+      property, when it would not normally do so (e.g., if no
+      significant change is made the object or the scheduleAgent is set
+      to client).  The property MUST NOT be preserved in the JSCalendar
```

Comment:
Above we wrap the value `none` in quotes, we should probably wrap `client` in quotes here too? Also should this mention `none` as well?